age/data/load/countries/germany.py

```diff
@@ -31,9 +32,8 @@
 ISO = 'DEU'
 
 class Germany(base.LoaderBase):
-    def __init__(self, reference_data):
+    def __init__(self):
```

Comment:
Please also remove from the usage in `age/data/load/generator.py`

Reply:
Sorry fixed.